templates/shared_services/airlock_notifier/app/AirlockNotifier/workflow.json

```diff
@@ -47,25 +47,32 @@
                   "event_value": {
                     "type": "string"
                   },
-                  "owners_emails": {
-                    "items": [
-                      {
-                        "type": "string"
-                      }
-                    ],
-                    "type": "array"
-                  },
                   "request_id": {
                     "type": "string"
                   },
-                  "researchers_emails": {
-                    "items": [
-                      {
-                        "type": "string"
-                      }
-                    ],
-                    "type": "array"
-                  },
+                  "notification_emails": {
+                    "type": "object",
+                    "properties": {
+                        "WorkspaceResearcher": {
```

Comment:
make it snake_cased to align with other attributes

Reply:
It's the name of the role assignment, So I think if someone will want to edit in the future it will be clearer to leave it as the role name without making it plural

Reply:
Made it snake_case thought to align